SvgImage.js

```diff
@@ -43,8 +43,8 @@ class SvgImage extends Component {
             scalesPageToFit={false}
             style={[
               {
-                width: 200,
-                height: 100,
+                width: props.style.width,
+                height: props.style.height,
                 backgroundColor: 'transparent',
               },
               props.style,
```

Comment:
We are passing `props.style` as second element in the array. width and height passed in props.style would override the default values in the above object.

PS: React Native defaults to showing image in 0x0 px if no width or height is specified. SInce this package intends to have feature parity with RN, we should probably set default width and height to 0 as well.